db/migrate/20170712171513_sites_cultivars_uniqueness.rb

```diff
@@ -1,5 +1,19 @@
 class SitesCultivarsUniqueness < ActiveRecord::Migration
-  def change
-    add_index :sites_cultivars, :site_id, unique: true
+  def up
+    this_hostid = Machine.new.hostid
+
+    execute %{
+      SELECT setval('sites_cultivars_id_seq', GREATEST(1, CAST(1e9 * #{this_hostid}::int AS bigint)), FALSE);
+
+      ALTER TABLE sites_cultivars ADD CONSTRAINT unique_site_id UNIQUE (site_id);
```

Comment:
I think the constraint should be `unique (site_id, cultivar_id)` to allow for the case where a site or plot is reused. Otherwise it may break on betydb.org